<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientChs</class>
 <widget class="QWidget" name="ClientChs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton{
	border-style:none;
	font: 12pt &quot;幼圆&quot;;
}
QToolButton:hover{background-color:rgb(201, 218, 255)
}

QLabel{
	
	font: 12pt &quot;幼圆&quot;;
}


QLineEdit{
	
	font: 12pt &quot;幼圆&quot;;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:rgb(0, 0, 0);
border-color:#FFFFFF;
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:1px;
border-left-width:0px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:rgb(0, 0, 0);
border-color:#FFFFFF;
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:1px;
border-left-width:0px;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="1.qrc">
     <normaloff>:/Image/jianshao.png</normaloff>:/Image/jianshao.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="icon">
    <iconset resource="1.qrc">
     <normaloff>:/Image/delete.png</normaloff>:/Image/delete.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Image/watercolor-4117017_1280_1.png);
	
	border-radius:15px;

</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>toolButton</zorder>
  <zorder>toolButton_3</zorder>
  <zorder>toolButton_4</zorder>
 </widget>
 <resources>
  <include location="1.qrc"/>
 </resources>
 <connections/>
</ui>
